tests/containers_test.py

```
@@ -65,6 +66,37 @@ def test_document_boolean_visibility_invalid(self) -> None:
+        assert container.name == "name"
+
+    def test_container_feature_append(self) -> None:
+        container = containers._Container(
+            ns="ns",
+            id="id",
+            target_id="target_id",
+            name="name"
+        )
+        feature = features._Feature(name="new_feature")
+        assert container.append(feature) is None
+        with pytest.raises(ValueError):
```

Ruff

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)